plugins/ipam/dhcp/dhcp_test.go

```diff
@@ -99,6 +110,10 @@ const (
 	pidfilePath  string = "/var/run/cni/dhcp-client.pid"
 )
 
+var socketPath string
+var tmpDir string
+var err error
+
 var _ = BeforeSuite(func() {
 	os.Remove(socketPath)
 	os.Remove(pidfilePath)
```

Comment:
I don't think we need the Remove() for socketPath and pidfilePath anymore, since we're calling RemoveAll on the directory they are in.  Same for AfterSuite()

Reply:
pidfilePath is (currently) in /var/run/cni, not /run/cni.  Should it be moved to /run/cni?